notifications/__init__.py

```diff
@@ -0,0 +1 @@
+"""Implement the notifications pipeline in its entirety."""
```

Comment:
All docstrings in this PR should have the double-quotes on their own lines.  So, this should be replaced with:
```
"""
Implement the notifications pipeline in its entirety.
"""
```
And please update all other docstrings similarly.  This is the convention we are using at edX.